drivers/counter/counter_smartbond_timer.c

```diff
@@ -150,14 +142,13 @@ static int counter_smartbond_start(const struct device *dev)
 				  TIMER2_TIMER2_CTRL_REG_TIM_EN_Msk |
```

Comment:
Registers access should take place after PM operations so the controller is resumed.

Reply:
Fixed